### PR TITLE
[DOC-3661] Harness module supported features - SMP

### PR DESCRIPTION
**Pull request by brian-f-harness**
Harness module supported features - SMP

For reviewers, a preview is available.

[What's supported](https://64f0f7308cf9c33c536e4ebd--harness-developer.netlify.app/docs/self-managed-enterprise-edition/smp-supported-platforms)

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/self-managed-enterprise-edition/smp-supported-platforms.md [hunk 1]
```diff
@@ -45,11 +58,11 @@ The following permissions are required:
 
 ## Supported Harness modules
```
Comment (schoudhury) [inline docs/self-managed-enterprise-edition/smp-supported-platforms.md:60]: don't see a reason to have this section now that we have the section at the top on this same page

File: docs/self-managed-enterprise-edition/smp-supported-platforms.md [hunk 2]
```diff
@@ -9,6 +9,19 @@ import Smp from '/docs/self-managed-enterprise-edition/shared/smp-supported-plat
 
 <Smp />
 
+## Harness module supported features
```
Comment (schoudhury) [inline docs/self-managed-enterprise-edition/smp-supported-platforms.md:12]: change this to "Supported modules and features"

File: docs/self-managed-enterprise-edition/smp-supported-platforms.md (hunk 1 above)
Comment (schoudhury) [reply docs/self-managed-enterprise-edition/smp-supported-platforms.md:60]: i mean remove this section altogether

**Review by schoudhury — APPROVED** [review]

File: docs/self-managed-enterprise-edition/smp-supported-platforms.md (hunk 2 above)
Comment (brian-f-harness) [reply docs/self-managed-enterprise-edition/smp-supported-platforms.md:12]: Updated

File: docs/self-managed-enterprise-edition/smp-supported-platforms.md (hunk 1 above)
Comment (brian-f-harness) [reply docs/self-managed-enterprise-edition/smp-supported-platforms.md:60]: removed

**Review by douglas-j-bothwell — DISMISSED** [review]

**Pull request merged**